### PR TITLE
github: Use main instead of master

### LINKED ISSUE #5084: (January 23, 2023) Please rename default branch from 'master' to 'main' per Khronos policy
To repository owners: please rename the default branch of this repository from 'master' to 'main', using the [Github renaming tool](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-branches-in-your-repository/renaming-a-branch). This request is per policy set by the Khronos Promoters in May 2021, to follow Github community practice for respectful naming.

The Github renaming tool sets up URL redirections and retargets outstanding pull requests, so the impact on repository users is minimal. The most visible issue is that people with local repo clones will probably want to rename their clone's 'master' branch, following the popup instructions that will be seen when browsing the github repository after the change; or just delete 'master' and pull the new 'main', if it's purely a tracker with no local content.

You may wish to coordinate with @outofcontrol if you are doing auto-updates from this repository to another location, whether via push/pull mirroring or other mechanisms. The redirects setup by Github should accommodate most such uses transparently, but it's still good practice.

Based on experience with other KhronosGroup repositories which have undergone this renaming already, this is a reasonable approach:

* Agree on a date for the renaming within the Working Group or other owners of the repository, and document that date here.
  * Date can be adjusted to avoid interaction with major releases in flight.
* Provide notice to repository users outside Khronos, insofar as possible (adding a comment in the repo README with the switchover date is one way).
* Once the renaming is done, edit the new 'main' branch to replace hardwired references to 'master' with (preferably) 'default branch' or 'main'.
* Update the repo README to note the change.

If you have questions or issues about this, please raise them on Khronos internal gitlab 'khronos-general' issue 106 if possible. If not possible, you can @-tag me here.

While we will not force any WG into acting precipitously, this is our agreed policy. Please try to accommodate renaming relatively soon.

Note that this issue is automatically generated, due to the large number of KhronosGroup repositories it's being raised in.

### PR DESCRIPTION
**Pull request by juan-lunarg**
closes #5084

### CODE REVIEW

File: BUILD.md [hunk 1]
```diff
@@ -364,9 +364,6 @@ Tested on OSX version 10.13.3
 
 Setup Homebrew and components
 
-- Follow instructions on [brew.sh](http://brew.sh) to get Homebrew installed.
-
-      /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
```
Comment (juan-lunarg) [inline BUILD.md:369]: Out of date instructions.

File: BUILD.md [hunk 2]
```diff
@@ -431,9 +428,6 @@ NOTE: To force the OSX version set the environment variable [MACOSX_DEPLOYMENT_T
...
```
Comment (juan-lunarg) [inline BUILD.md:436]: Out of date instructions.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -104,10 +104,6 @@ that to be accepted into the repository, the pull request must [pass all tests](
...
 
-#### **GitHub Cloud CI Testing**
-Pull Requests to GitHub are tested in the cloud on Linux and Windows VMs. The Linux VMs use [Github Actions](https://github.com/KhronosGroup/Vulkan-ValidationLayers/actions) with the sequence of commands driven by the [ci_build.yml](https://github.com/KhronosGroup/Vulkan-ValidationLayers/blob/master/.github/workflows/ci_build.yml) file. The Windows VMs use [AppVeyor](https://ci.appveyor.com/project/Khronoswebmaster/vulkan-validationlayers/branch/master) with the sequence of commands driven by the [.appveyor.yml](https://github.com/KhronosGroup/Vulkan-ValidationLayers/blob/master/.appveyor.yml) file.
-
-The Linux testing includes iterating on all of the validation layer tests over multiple [different device](https://github.com/KhronosGroup/Vulkan-ValidationLayers/tree/master/tests/device_profiles) profiles using the [Profile Layer](https://github.com/KhronosGroup/Vulkan-Profiles) in combination with the [mock icd](https://github.com/KhronosGroup/Vulkan-Tools/tree/master/icd). This is a fast way to simulate testing across different devices. Any new tests must pass across all device profiles.
```
Comment (juan-lunarg) [inline CONTRIBUTING.md:110]: Removed entirely since writing about our CI ages poorly and CI is decently self evident.

**Review by jeremyg-lunarg — APPROVED** [review]

File: tests/layer_validation_tests.cpp [hunk 4]
```diff
@@ -3604,7 +3604,6 @@ void addFullTestCommentIfPresent(const ::testing::TestInfo &test_info, std::stri
     }
 }
 
-// Inspired by https://github.com/google/googletest/blob/master/googletest/docs/AdvancedGuide.md
```
Comment (juan-lunarg) [inline tests/layer_validation_tests.cpp:3607]: deadlink

**Review by juan-lunarg — COMMENTED** [review]
Overall cleanup to the documentation / scripts regarding anything that said `master` when I searched in vscode.

**Pull request merged**